build-logic/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/UpdateAgpVersions.kt

```diff
@@ -31,6 +32,7 @@ import javax.xml.parsers.DocumentBuilderFactory
  * Fetch the latest AGP versions and write a properties file.
  * Never up-to-date, non-cacheable.
  */
+@DisableCachingByDefault(because = "Not worth caching")
```

Comment:
I think here the reason is more like it cannot be cached, as it changes in-place, no?

Reply:
It loads data from the internet and then writes the file. This is not about changing things in-place.